.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: Codecov
```

Comment:
**nit**:  We should rename this to publish package?

Reply:
there is two jobs in this workflow the 1st one will test and upload test coverage to codecov and after this publish will trigger.

Reply:
Changed to `Publish Package`